### ISSUE TITLE
Generate as extension methods?

### ISSUE THREAD

**Issue by aeslinger0** [cezarypiatek/MappingGenerator#43]
Looks like a great tool. I think a good enhancement would be to generate the mapping code as an extension method.

`userEntity.ToUserDTO()`

**Comment by cezarypiatek** [comment]
Hi

Thanks for suggestion. I have a few questions on start: where the extension method should me added? Should I generate a separated static class? How the class should be named?

**Comment by aeslinger0** [comment]
Actually, I should be able to use [your first example](https://github.com/cezarypiatek/MappingGenerator#generate-mapping-method-body) to generate the extension method body and then I can just add `this` before the parameter manually. But I'm not able to get your extension to work. I'm using VS Community 2017. Is that supported?

**Comment by cezarypiatek** [comment]
I've just tested it on extension method and it works. I'm using Visual Studio 15.8.4. Make sure that you restarted VS after extension installation. Community version should be supported. 

![image](https://user-images.githubusercontent.com/7759991/46517846-8c916a80-c871-11e8-8399-5b028920384d.png)

Please let me know if the restart didn't help (the concrete version of VS would be helpful)

**Comment by aeslinger0** [comment]
I tried uninstalling and reinstalling the extension, restarting VS, and rebooting my computer. I was using VS 15.5.1. I updated to 15.8.4 and now it works! Not sure if it was that specific version of VS that wasn't working or if something got corrupted and updating fixed it. Either way, I'm ready to start killing off AutoMapper! I'll let you know if I run into any issues with the extension. Thanks!
